package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "atomizer",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
We can bump it now, but I'd hold the publish after we implement pseudo classes and breakpoints. Thoughts?